site/en/docs/devtools/recorder/index.md

```diff
@@ -105,27 +113,55 @@ Let's walk through the options to edit the user flows.
 
 Let's walk through the options to edit the steps within a workflow.
 
-1. Expand each step to see the details of the action. For example, expand the *Click Element "Cappucino"* step.
-  {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/ADbB4FFeiBBZqbxKRw0s.png", alt="In the recorder panel, the Cappucino element has been expanded to reveal type, target, selectors, offset X, and offset Y.", width="800", height="663" %}
-2. The step above shows 2 **selectors**. The **Recorder** will automatically detect 2 selectors for most of the steps by default.
-    - When replaying the user flow, the **Recorder** will try to query the element with one of the selectors by sequence.
-    - For example, if the **Recorder** successfully queries the element with the first selector, it will skip the second selector and proceed to the next step.
-3. You can add or remove any selectors. For example, you can remove the *selector #2* because just `aria/Cappucino` is sufficient in this case. Hover over the *selector #2* and click on **-** to remove it.
+### Expand steps {: #expand-step }
+
+1. Expand each step to see the details of the action. For example, expand the *Click Element "Cappuccino"* step.
+  {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/WUIZb8eMWfcPZHPyQ5C5.png", alt="In the recorder panel, the Cappuccino element has been expanded to reveal type, target, selectors, offset X, and offset Y.", width="800", height="773" %}
+1. The step above shows two **selectors**. For more information, see [Selector types](#selector-types).
+   When replaying the user flow, the **Recorder** tries to query the element with one of the selectors by sequence.
+   For example, if the **Recorder** successfully queries the element with the first selector, it will skip the second selector and proceed to the next step.
+1. You can add or remove any selectors. For example, you can remove the *selector #2* because just `aria/Cappuccino` is sufficient in this case. Hover over the *selector #2* and click on **-** to remove it.
     {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/qEat3jT1Og68urhmSkef.png", alt="The DevTools recorder panel shows an option to remove a selector", width="800", height="663" %}
-4. The **selector** is editable too. For example, if you want to select *Mocha* instead of *Cappucino*, you can edit the selector value to *aria/Mocha* instead.
+1. The **selector** is editable too. For example, if you want to select *Mocha* instead of *Cappuccino*, you can edit the selector value to *aria/Mocha* instead.
   {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/OdihJ4BROOXZ93eidr4b.png", alt="edit a selector", width="800", height="663" %} 
-5. [Replay](#replay) the flow now, it should select *Mocha* instead of *Cappucino*.
-6. Try to edit other step properties such as **type**, **target**, **value** and more.
+1. [Replay](#replay) the flow now, it should select *Mocha* instead of *Cappuccino*.
+1. Try to edit other step properties such as **type**, **target**, **value** and more.
```

Comment:
There are two ways of editing the value, user can

a. edit the Mocha, or
b. use the selector picker to pick the selector on the screen. (I demo this in the video but not in this doc.)

Please add a step to explain b)